qiskit/providers/ibmq/ibmqbackendservice.py

```diff
@@ -369,7 +369,7 @@ def retrieve_job(self, job_id: str) -> IBMQJob:
         try:
             job_info = self._provider._api.job_get(job_id)
         except ApiError as ex:
-            raise IBMQBackendApiError('Failed to get job "{}": {}'
+            raise IBMQBackendApiError('Failed to get job {}: {}'
```

Comment:
This should also have `from ex`. Can you check if there are others that should be chained or not chained (`raise from None`)?